pkg/apis/tensorflow/v1alpha1/defaults.go

```diff
@@ -24,7 +24,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 }
 
 // SetDefaultsTFJob initializes any uninitialized values to default values
-func SetDefaults_TFJob(obj *TFJob) {
+func SetDefaultsTFJob(obj *TFJob) {
```

Comment:
I think we should not update the code here sine it is the convention for k8s defaulter-gen.

Reply:
I think we could revert the change here and set the file as ignored files in linter_config.json

Reply:
okay! sure I will revert the last commit and add that file in linter_config.json

Reply:
Thanks! Sorry for the late reply, or we could avoid the problem.

Thanks for your contribution, again.